packages/graphiql/src/index.tsx

```diff
@@ -61,7 +61,7 @@ const theme: MantineThemeOverride = {
   },
 };
 
-function fetcher(params: FetcherParams): Promise<SyncExecutionResult> {
+async function fetcher(params: FetcherParams): Promise<SyncExecutionResult> {
```

Comment:
The `async` keyword should not be necessary here.  You only need to add `async` if you use the `await` keyword in the function body.